### DISCUSSION TITLE
Customize font rendering

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#94] [Ideas]
On Lite XL, I could customize font antialiasing and hinting mode on Font Selector dialog. Go to Settings -> Editor to customize Code Font. Double click on an existing font or add a new font will open the Font Selector dialog. Just open Lite XL to see. I disliked the out of the box blurry font of Lite XL and ecode. How could I customize font rendering like on Lite XL? Thank you.

**Comment by SpartanJ** [comment]
Oh, that isn't possible yet. I kind of forgot about it. What kind of hinting are you using? ecode does not support [sub-pixel hinting](https://github.com/SpartanJ/ecode#current-limitations) yet, it uses grayscale hinting by default. I don't like subpixel hinting, and it's slowly dying with hidpi screens. Can you let me know what kind of hinting are you using and how it looks in Lite-XL (send me a screenshot, please)? Thanks

**Reply by ghost** [reply]
> Can you let me know what kind of hinting are you using and how it looks in Lite-XL (send me a screenshot, please)? Thanks

I have a better solution. What about you download and open Lite XL to see it yourself?

**Reply by ghost** [reply]
Lite XL by default uses subpixel antialiasing and hinting slight. I usually changed hinting to full to have the font less blurry.

**Reply by SpartanJ** [reply]
Ok, I know the options Lite-XL has. I needed to know what you used. Sub-pixel hinting is tricky because it requires some rendering tricks that I would prefer to avoid. The hinting levels are easier to make available, but ecode use full at grayscale. Are you using an low pixel density screen? Can you show me a comparison of what you like using the same font? Sometimes it's not tha obvious since it depends on the font size used.

**Reply by ghost** [reply]
Increase the font size solved the problem. Thank you.
